dcos_openvpn/cert.py

```diff
@@ -6,10 +6,18 @@
 import re
 import subprocess
 
-def path(name):
+def key_path(name):
     return os.path.join(os.environ.get("EASYRSA_PKI", ""),
         "private/{0}.key".format(name))
 
+def req_path(name):
+    return os.path.join(os.environ.get("EASYRSA_PKI", ""),
```

Comment:
Let's factor this out into a `def create_path(file_path)` method that is called like `create_path("reqs/{0}.req".format())`.